content/main/docs/serverless-workflows/troubleshooting.md

```diff
@@ -94,3 +94,16 @@ a CA which is not available to the workflow. The error in the workflow pod log u
 
 **Solution:**
 When deploying a workflow in a namespace other than the one where Sonataflow services are running (e.g., sonataflow-infra), there are essential steps to follow to enable persistence and connectivity for the workflow. See the following [steps](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/docs/main/README.md#additional-workflow-namespaces).
+
+### Problem: sonataflow-platform-data-index-service pods can't connect to the database on startup
+
+First, check if the PostgreSQL pod is still starting. If it is, allow some time for the pod to become fully operational before the DataIndex and JobService pods are ready.
+
+If the PostgreSQL pod is running but the issue persists, verify that the `sonataflow-infra` namespace has the required label `rhdh.redhat.com/workflow-namespace`. Missing this label can cause connectivity issues.
```

Comment:
May be worthy to give the command to check the rules applied by the NP:
`oc get networkpolicies.networking.k8s.io -n sonataflow-infra allow-rhdh-to-sonataflow-and-workflows -o yaml`?
The only drawbacks of doing so is if the name of the NP changes, we have to remember to update it here too
And no need to provide the output of the command here, just give it here so if the user wants to check it deeper, (s)he has all the inputs